tests/io/v0_0_0/test_write.py

```diff
@@ -180,3 +181,54 @@ def test_checksum_on_write(self):
                 loaded = slicedimage.Reader.parse_doc(basename, baseurl)
 
                 loaded.tiles()[0]._load()
+
+    def test_write_tiff(self):
+        image = slicedimage.TileSet(
+            ["x", "y", "ch", "hyb"],
+            {'ch': 2, 'hyb': 2},
+            (100, 100),
+        )
+
+        for hyb in range(2):
+            for ch in range(2):
+                tile = slicedimage.Tile(
+                    {
```

Comment:
similarly, kwargs would make this clearer. 

I also think we should move towards using "real" coordinates, i.e. non-overlapping or barely-overlapping coordinate sets in place of the existing coordinates that are identical for each slice. 

That will support tests of multi-fov `Experiment` workflows.